truck_inspection_client/config/parameters.yaml

```diff
@@ -0,0 +1,37 @@
+# Vehicle VIN number - 17 alphanumeric char
+vin_number: 1FUJGHDV0CLBP8834
+
+# License plate number - 8 alphanumeric char
+license_plate: DOT-10003
+
+# Carrier Name - 20 alphanumeric char
+carrier_name: Silver Truck FHWA TFHRC
+
+# Carrier ID - (USDOT + space + 7digits)
+carrier_id: USDOT 0000001
+
+# Weight: ###,### lbs. (6 number format)
+weight: 100000
+
+# ADS Software Version: CARMA Platform v#.#.# (25characters)
+ads_software_version: CARMA Platform v3.3.0
```

Comment:
Per John, he wants this from the CARMAConfig version like the UI

Reply:
Looks like that is something not doable from C++ node.

Reply:
Fixed as discussed.